content/en/contribute/code/hall-of-fame.md

```diff
@@ -15,7 +15,6 @@ Thank you to everyone who has contributed to the CHT codebase over the years! To
 - [CHT Core](https://github.com/medic/cht-core/graphs/contributors)
 - [CHT Docs](https://github.com/medic/cht-docs/graphs/contributors)
 - [CHT Interoperability](https://github.com/medic/cht-interoperability/graphs/contributors)
-- [CHT Pipeline](https://github.com/medic/cht-pipeline/graphs/contributors)
```

Comment:
This one might still be useful though I agree with dropping the naming. Perhaps the text of all these links should be the repo name, not the brand name?